apps/frontend/src/modules/projects/slices/actions.ts

```diff
@@ -30,6 +35,20 @@ const loadAll = createAsyncThunk<
 >(`${sliceName}/load-all`, async (query, { extra }) => {
 	const { projectApi } = extra;
 
+	return await projectApi.getAll({
+		limit: INFINITE_SCROLL_LOAD_COUNT,
+		name: query ?? "",
+		start: DEFAULT_START,
```

Comment:
We need to pass page, pageSize from the component. Let's create an additional hook, like useInfiniteScroll, which will do something similar to usePagination, but much simpler - without query params, and without an ability to change pageSize